src/v0/destinations/am/transform.js

```diff
@@ -973,12 +989,22 @@ const processRouterDest = async inputs => {
           input.destination
         );
       } catch (error) {
-        const errRes = generateErrorObject(error);
-        return getErrorRespEvents(
+        const errObj = generateErrorObject(error);
+        const resp = getErrorRespEvents(
           [input.metadata],
-          error.status || 400,
-          error.message || "Error occurred while processing payload.",
-          errRes.statTags
+          errObj.status,
+          errObj.message,
+          errObj.statTags
+        );
+
+        errNotificationClient.notify(
```

Comment:
does it notify all the errors or some kind of filtering logic in implemented in `errNotificationClient.notify`

Reply:
Yes, it has filtering logic to ignore the known error types.